tests/values.yaml

```diff
@@ -1,11 +1,11 @@
 nodes:
   - name: 'node-1'
     msp_id: 'MyPeer1MSP'
-    address: 'http://substra-backend.node-1'
-    user: 'node-1'
-    password: 'node-1pwd'
+    address: 'http://substra-backend.node-1.com'
+    user: 'substra'
+    password: 'p@$swr0d44'
   - name: 'node-2'
     msp_id: 'MyPeer2MSP'
-    address: 'http://substra-backend.node-2'
-    user: 'node-2'
-    password: 'node-2pwd'
+    address: 'http://substra-backend.node-2.com'
+    user: 'substra'
+    password: 'p@$swr0d44'
```

Comment:
I'm just leaving a comment to reference that the change from 2pwd to 1pwd is also discussed in https://github.com/SubstraFoundation/substra-backend/pull/1#discussion_r340105302

Reply:
updated